__tests__/app.test.js

```diff
@@ -116,11 +116,48 @@ describe("GET/api/categories", () => {
   .expect(200)
   .then((response)=>{
     
-    console.log(response.body, 'inside test')
     expect(response.body.comments).toStrictEqual([])
   })
  })
 
+ describe('POST api/reviews/:review_id/comments',()=>{
+    test('400: not found', ()=>{
+      return request(app)
+      .post('/api/reviews/40/comments')
+      .expect(404)
+      .then((response)=>{
+        const msg = response.body.msg
+        expect(msg).toBe("not found")
+      })
+    })
+    test('400:invalid review_id',()=>{
+      return request(app)
+      .get('/api/reviews/orange/comments')
+      .expect(400)
+      .then((response)=>{
+        const msg = response.body.msg
+        expect(msg).toBe("bad request")
+      })
+ })
+ 
+      test.only('should Responds with the posted comment',()=>{
```

Comment:
Please remove .only to allow all the tests to run and resubmit your pr.

Reply:
sorry i just forgot it now it's done